coral/src/app/components/Form.tsx

```diff
@@ -464,6 +463,11 @@ const SubmitButtonMemo = memo(
   }
 ) as typeof _SubmitButton;
 
+/*** Pass `loading` property to button when
```

Comment:
Nitpicky and pedantic, but I think this comment is a little misleading, because the conditions for `isSubmitting` to be correctly updated are:
- when `onSubmit` **returns** a `Promise`
- and that promise has a `catch` which **calls** `Promise.reject()` (for the cases when that returned Promise throws an error)

We could link to this comment: https://github.com/Aiven-Open/klaw/issues/1439#issuecomment-1699202614

Reply:
Will update!

Reply:
https://github.com/Aiven-Open/klaw/pull/1743/commits/7cd9618d831484a51c18bd53a5f4e9e62b6a438b